### PR TITLE
feat: test rock class

### PR DESCRIPTION
**Pull request by i-chvets**
Summary of changes:
- Added TestRock class.

### CODE REVIEW

File: src/charmed_kubeflow_chisme/rock/_test_rock.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+class TestRock:
+    """Test class for ROCKs."""
+
+    def __init__(self):
+        """Initializes and return TestRock instance."""
+        # no initialization is currently needed
+
+    def get_rock_image_name_from_rockcraft(self, file):
+        """Read ROCK information and return ROCK image name."""
```
Comment (ca-scribner) [inline src/charmed_kubeflow_chisme/rock/_test_rock.py:16]: Maybe:
```suggestion
    def get_rock_image_name_from_rockcraft(self, file: str):
        """Reads a rockcraft.yaml file and returns the ROCK image name."""
```
?

On first read, I wasn't sure what `file` should be.  Could also rename to `filename` so it is clear, but having type of `str` makes that pretty clear too

File: src/charmed_kubeflow_chisme/rock/README.md
Comment (ca-scribner) [inline src/charmed_kubeflow_chisme/rock/README.md:1]: Can we also add a bullet in [the top level readme](https://github.com/canonical/charmed-kubeflow-chisme/blob/kf-2316-feat-test-rock/README.md)?

Comment (i-chvets) [reply src/charmed_kubeflow_chisme/rock/README.md:1]: Done.

**Review by ca-scribner — APPROVED** [review]
lgtm!  ty!

**Pull request merged**